### ISSUE TITLE
Controlling display of feedback module

### ISSUE THREAD

**Issue by toribr** [cityofaustin/techstack#3717]
### As a [user type] I want [capability] so that [benefit].

**Is your feature request related to a problem? Please describe.**
In some cases, a feedback module detracts from the user experience. Currently we know that this is true of forms. But there may be other cases (the home page comes to mind) where we would prefer to repress the display of the feedback module.

**Describe the solution you'd like**
Perhaps a way to repress by url or by url structure, ie, all pages below /form/

**Describe alternatives you've considered**

**Additional context**

**Comment by desigonz** [comment]
Are you thinking this is something we'd control on the author interface?

**Comment by toribr** [comment]
Yes, likely a super admin thing, where all pages with the module repressed can be viewed.
